tests/dummy/config/environment.js

```diff
@@ -1,9 +1,13 @@
+/* eslint-env node */
 module.exports = function (environment) {
   var ENV = {
     APP: {},
-    rootURL: '/',
```

Comment:
This should stay.   Similar entry in test configuration should be removed though